packages/inter-protocol/test/vaultFactory/vault-contract-wrapper.js

```diff
@@ -79,31 +80,41 @@ export async function start(zcf, privateArgs, baggage) {
     return floorDivideBy(getAmountOut(quoteAmount), LIQUIDATION_MARGIN);
   };
 
-  const reallocateWithFee = (fee, wanted, seat, ...otherSeats) => {
-    const toMint = AmountMath.add(wanted, fee);
+  /** @type {MintAndTransfer} */
+  const mintAndTransfer = (mintReceiver, toMint, fee, nonMintTransfers) => {
+    const kept = AmountMath.subtract(toMint, fee);
     runMint.mintGains(harden({ Minted: toMint }), stage);
+    /** @type {import('@agoric/zoe/src/contractSupport/atomicTransfer.js').TransferPart[]} */
+    const transfers = [
+      ...nonMintTransfers,
+      [stage, vaultFactorySeat, { Minted: fee }],
+      [stage, mintReceiver, { Minted: kept }],
+    ];
     try {
-      vaultFactorySeat.incrementBy(stage.decrementBy(harden({ Minted: fee })));
-      seat.incrementBy(stage.decrementBy(harden({ Minted: wanted })));
-      zcf.reallocate(vaultFactorySeat, stage, seat, ...otherSeats);
+      atomicRearrange(zcf, harden(transfers));
     } catch (e) {
+      console.error('mintAndTransfer caught', e);
       stage.clear();
       vaultFactorySeat.clear();
+      // Make best efforts to burn the newly minted tokens, for hygiene.
+      // That only relies on the internal mint, so it cannot fail without
+      // there being much larger problems. There's no risk of tokens being
+      // stolen here because the staging for them was already cleared.
```

Comment:
There's no staging remaining, right?  What is this referring to?

Reply:
This was just copied over. But that means I should revert the change here and go update vaultDirector and stakeFactory to not talk about staging.